src/model/row.py

```diff
@@ -0,0 +1,54 @@
+import uuid
+import os
+import json
+import pathlib
+from model.file_mode import *
+
+
+class Row:
+    def __init__(self, table, data):
+        self.table = table
+        self.primary_key = self.__get_primary_key__(data.get(table.get_primary_key()))
+        data[table.get_primary_key()] = self.primary_key
+        self.data = data
+
+    def __get_primary_key__(self, primary_key):
+        if primary_key is not None:
+            return primary_key
+        path = self.table.get_path()
+        while primary_key is None and os.path.exists(path):
+            primary_key = uuid.uuid1().node
+            path = self.get_path(primary_key)
+        return primary_key
+
+    def get_path(self, primary_key=None):
+        primary_key = primary_key if primary_key else self.primary_key
+        return os.path.join(self.table.__get_data_path__(), "{}.json".format(primary_key))
+
+    @staticmethod
+    def create(path, data, can_overwrite):
```

Comment:
Create gives me an impression creating a new rows.

Reply:
This method has just two lines. Move it to serialize

Reply:
you asked me to do this :
class Row {
  Row(table, data) {
      self.primary_key = self.__get_primary_key()
   }
}

def create(). <--- static 
def delete(self) 
def __get_path(self)
row.serialize()

Reply:
i will move it